terraform/environments/electronic-monitoring-data/server_backups.tf

```diff
@@ -96,6 +96,17 @@ resource "aws_vpc_security_group_ingress_rule" "db_ipv4_lb" {
 
   cidr_ipv4 = "209.35.83.77/32"
 }
+
+resource "aws_vpc_security_group_ingress_rule" "db_glue" {
+  security_group_id = aws_security_group.db.id
+  description       = "glue"
+  ip_protocol       = "tcp"
+  from_port         = 0
+  to_port           = 65535
+
+  cidr_ipv4 = "152.37.111.98/32"
```

Comment:
is this your ip?

Reply:
my home one ye - im in 10SC today tho haha